src/main/resources/db/migration/v0.21.0.2__delete_duplicated_slot_annotation_table_indexes.sql

```diff
@@ -0,0 +1,4 @@
+DROP INDEX genefullname_singleallele_curie_index;
+DROP INDEX genesymbol_singleallele_curie_index;
+DROP INDEX genesynonym_singleallele_curie_index;
+DROP INDEX genesystematicname_singleallele_curie_index;
```

Comment:
@markquintontulloch are these duplicated on the allele slot annotations also. I thought that was where I saw them.

Reply:
@oblodgett couldn’t see any on the allele slot annotations